release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -541,6 +541,12 @@ With this release, MetalLB includes a new field for the Border Gateway Protocol
 You can use the `dynamicASN` field to detect the Autonomous System Number (ASN) to use for the remote end of a BGP session.  
 This is an alternative to explicitly setting an ASN in the `spec.peerASN` field.
 
+[id="ocp-release-notes-networking-sr-iov-rdma-cni_{context}"]
+==== Configuring an RDMA subsytem for SR-IOV 
+
+With this release, you can configure an RDMA Container Network Interface (CNI) on Single Root I/O Virtualization (SR-IOV) to enable high-performance, low-latency communication between containers. 
+When you combine RDMA with SR-IOV, you provide a mechanism to expose hardware counters of Mellanox Ethernet devices for use inside Data Plane Development Kit (DPDK) applications.
```

Comment:
```suggestion
With this release, you can configure an Remote Direct Memory Access (RDMA) Container Network Interface (CNI) on Single Root I/O Virtualization (SR-IOV) to enable high-performance, low-latency communication between containers. 
When you combine RDMA with SR-IOV, you provide a mechanism to expose hardware counters of Mellanox Ethernet devices to use inside Data Plane Development Kit (DPDK) applications.
```

Reply:
Thanks for the review @xenolinux 
I'm not sure about changing "for" to "to" on line 548. It would need to be "to be used..." I think, which is passive voice....

Reply:
Agree. However, I feel the sentence reads off with `for use...`

Reply:
If there is no subject to mention, then I'd go with the passive voice